packages/wallets/src/components/Base/WalletLink/WalletLink.tsx

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import { getStaticUrl } from '../../../helpers/urls';
 import './WalletLink.scss';
 
+type TVariant = 'bold' | 'normal';
+
 interface LinkProps {
     children?: React.ReactNode;
     href?: React.AnchorHTMLAttributes<HTMLAnchorElement>['href'];
     staticUrl?: React.AnchorHTMLAttributes<HTMLAnchorElement>['href'];
+    variant?: TVariant;
 }
 
-const WalletLink: React.FC<LinkProps> = ({ children, href, staticUrl }) => (
+const WalletLink: React.FC<LinkProps> = ({ children, href, staticUrl, variant = 'normal' }) => (
     <a
-        className='wallets-link'
+        className={`wallets-link__variant--${variant}`}
```

Comment:
```suggestion
        className={`wallets-link wallets-link__variant--${variant}`}
```
We still need the base styles

Reply:
Updated